packages/geo/src/slug.ts

```diff
@@ -63,7 +69,20 @@ export const LocationSlug = {
     };
   },
 
-  cameraStr(loc: LonLatZoom): string {
+  /**
+   * Truncate a bearing and pitch
+   *
+   * @param loc location to truncate
+   */
+  truncateBearingPitch(loc: LonLatZoom): Partial<LonLatZoom> {
+    let bearing;
+    let pitch;
+    if (loc.bearing) bearing = Number(loc.bearing.toFixed(LocationSlug.BearingFixed));
+    if (loc.pitch) pitch = Number(loc.pitch.toFixed(LocationSlug.PitchFixed));
+    return { bearing, pitch };
```

Comment:
Could avoid the `let` with something like:

```suggestion
return {
 bearing: loc.bearing ? Number(loc.bearing.toFixed(LocationSlug.BearingFixed)) : undefined,
 pitch: loc.pitch ? Number(loc.pitch.toFixed(LocationSlug.PitchFixed)) : undefined
 }
```